### ISSUE TITLE
Crop Beta City yeg photo

### ISSUE THREAD

**Issue by MandyMeindersma** [devedmonton/DES-Website#219]
![image](https://github.com/devedmonton/DES-Website/assets/17459171/bca7e9cc-073e-4946-9ef2-99b3c013d5d9)

When you are viewing in mobile the styling looks off so just crop it so it alligns

**Comment by DerrykBoyd** [comment]
Maybe a better solution here is to make the Logo image optional? Seems like in this case we don't need both the Logo and the header.

**Comment by MandyMeindersma** [comment]
I think Beta city is the only one without a cool photo though, maybe the hardware meetup too. I think it might look a little weird if they were the odd ones out?

**Comment by DerrykBoyd** [comment]
> I think Beta city is the only one without a cool photo though, maybe the hardware meetup too. I think it might look a little weird if they were the odd ones out?

Yeah you are right, I tried without and looked odd. Created #225 to trim the whitespace off the logo sides.
